RPresto.Rproj

```diff
@@ -5,8 +5,13 @@ SaveWorkspace: No
 AlwaysSaveHistory: Default
 
 EnableCodeIndexing: Yes
+UseSpacesForTab: Yes
```

Comment:
Could you remove the RPresto.Rproj changes? I'd prefer to keep the context of the PR limited to session parameters.